src/ui/public/courier/search_source/search_source.js

```diff
@@ -181,211 +168,134 @@ export function SearchSourceProvider(Promise, Private, config) {
      * PUBLIC API
      *****/
 
+    setData = newData => {
```

Comment:
I'm sorry, but I really don't like needless function arrows in class bodies. They're useful when you need to bind a handler to the `this` context, but using them everywhere is unnecessary.